engine/xml-config/src/main/resources/configuration.xsd

```diff
@@ -516,6 +516,11 @@
 							</sequence>
 						</complexType>
 					</element>
+					<element name="username" minOccurs="0" maxOccurs="1" type="string" />
+					<element name="password" minOccurs="0" maxOccurs="1" type="string" />
+					<element name="ssl" minOccurs="0" maxOccurs="1" type="boolean" />
+					<element name="keystorePath" minOccurs="0" maxOccurs="1" type="string" />
```

Comment:
Our convention for naming XML nodes is to use dash instead of camelCase. So this one should be "keystore-path" and below "keystore-password".